src/routes/api/tripRoute.js

```diff
@@ -239,4 +240,62 @@ tripRouter.post('/twoWay',
  *
  */
 
+tripRouter
+  .put(
+    '/edit/:id',
+    verifyToken,
+    isUserVerified,
+    Validate.isIDInteger(),
+    checkInputDataError,
+    Validate.editRequest(),
+    checkInputDataError,
+    Conflict.isTripRequestFound,
+    IsOwnerOfTrip,
+    IsTripApproved,
+    ValidateTrip.checkIfOriginDestinationExists,
+    ValidateTrip.checkIfOriginSameAsDestination,
+    ValidateTrip.checkMultiCityForSimilarRequests,
+    ValidateTrip.checkForSimilarRequests,
+    TripController.editTrip
+  );
```

Comment:
@NiyoEric  I'd recommend sticking to one convention. If you are destructuring please ensure it is destructured through and through